lua/telescope/previewers.lua

```diff
@@ -186,6 +200,21 @@ previewers.cat = defaulter(function(opts)
       }
     end,
 
+    send_input = function(self, input)
+      termcode = vim.api.nvim_replace_termcodes(input, true, false, true)
+      if self.state.termopen_id then
+        pcall(vim.fn.chansend, self.state.termopen_id, termcode)
+      end
+    end,
+
+    scroll_fn = function(self, direction)
```

Comment:
Do you think it'd be possible to add something like "count", similar to `v:count`? so someone can do `5<c-d>` or `scroll_fn(5)` to scroll five lines?

Reply:
I've changed that to using direction as count. So right now the default scrolling up is `scroll_fn(30)` and `scroll_fn(-30)`.